### PR TITLE
Change `plz run sequential` to have the same output flag as parallel

### PR DESCRIPTION
**Pull request by peterebden**
It would be nicer for people if these two mirror one another.

### CODE REVIEW

**Review by Tatskaari — APPROVED** [review]

File: src/run/run_step.go [hunk 1]
```diff
@@ -89,15 +71,35 @@ func Parallel(ctx context.Context, state *core.BuildState, labels []core.Annotat
 	return 0
 }
 
+// runWithOutput runs a subprocess with the given output mechanism.
+func runWithOutput(ctx context.Context, state *core.BuildState, label core.AnnotatedOutputLabel, args []string, output ProcessOutput, remote, env, detach, inTmp bool, dir string) error {
```
Comment (tiagovtristao) [inline src/run/run_step.go:75]: Much better having this logic here.

**Review by tiagovtristao — APPROVED** [review]
`GroupImmediate` should yield the same results as `Default` for `sequential`, but I guess this isn't an issue. None of the options are mutual exclusive, so I guess it makes sense for now to have both subcommands aligned. Thanks.

**Comment by peterebden** [comment]
> `GroupImmediate` should yield the same results as `Default` for `sequential`, but I guess this isn't an issue. None of the options are mutual exclusive, so I guess it makes sense for now to have both subcommands aligned. Thanks.

I think `GroupImmediate` adds a heading with the label of each target that's being run? So it's slightly different - but agreed the difference is much less significant than for `parallel`.

**Review by samwestmoreland — APPROVED** [review]

**Comment by tiagovtristao** [comment]
> > `GroupImmediate` should yield the same results as `Default` for `sequential`, but I guess this isn't an issue. None of the options are mutual exclusive, so I guess it makes sense for now to have both subcommands aligned. Thanks.
> 
> I think `GroupImmediate` adds a heading with the label of each target that's being run? So it's slightly different - but agreed the difference is much less significant than for `parallel`.

Ah yes, it does that, so you can say they are not exactly the same :-)

**Pull request merged**